crates/core/src/expr/statements/info.rs

```diff
@@ -161,7 +161,7 @@ impl InfoStatement {
 						"apis".to_string() => process(txn.all_db_apis(ns, db).await?),
 						"analyzers".to_string() => process(txn.all_db_analyzers(ns, db).await?),
 						"buckets".to_string() => process(txn.all_db_buckets(ns, db).await?),
-						"functions".to_string() => process(txn.all_db_functions(ns, db).await?),
+						"functions".to_string() => process(txn.combined_db_functions(ns, db).await?),
```

Comment:
Aren't they still "all_db_functions" ? It refers to all "DefineFunctionStatement". I don't think we need to change the prefix